README.md

```diff
@@ -5,12 +5,37 @@
 
 recode is a utility-focused fabric mod for [DiamondFire](https://mcdiamondfire.com/home/) to make development more fun. It is the official successor to [CodeUtilities](https://github.com/CodeUtilities/CodeUtilities-2.0), from which it was cloned.
 
-You can find the mod jar [here](https://modrinth.com/mod/recode/versions). Future plans include removing bloat/unnecessary features, improving existing code, adding new features, and migrating to Kotlin.
+Future plans include removing bloat/unnecessary features, improving existing code, adding new features, and migrating to Kotlin.
 
 Feel free to join the recode Discord server: https://discord.gg/GWxWtcwA2C
 
 <!-- modrinth_exclude.start -->
-Or check the Modrinth page: https://modrinth.com/mod/recode
+## Installation
+
+### Modrinth or GitHub releases (recommended)
+
+Click [here](https://modrinth.com/mod/recode) to get to the Modrinth page for the mod. Here you can click on the latest featured version to the left, and then just press the Download button.
+
+Alternatively, click [here](https://github.com/homchom/recode/releases) to get to the latest release of the mod, then navigate to Assets, and there you can find the mod's .jar file.
+
+### Building manually
```

Comment:
This is generally not something we want to recommend as in development versions can be very unstable and most normal users should not be guided to building an application from source.